packages/dev/core/src/Engines/thinEngine.ts

```diff
@@ -407,7 +407,15 @@ export class ThinEngine {
     protected _creationOptions: EngineOptions;
     protected _audioContext: Nullable<AudioContext>;
     protected _audioDestination: Nullable<AudioDestinationNode | MediaStreamAudioDestinationNode>;
-
+    public _glSRGBExtensionValues: {

```

Comment:
I think it needs a `/** @internal */` prefix.

Reply:
Done :-)